apps/site/src/app/(home)/components/article-tile/article-tile.component.tsx

```diff
@@ -1,19 +1,31 @@
 'use client';
 
 import { ArrowRightIcon } from '@westpac/ui/icon';
+import { BREAKPOINTS } from '@westpac/ui/themes-constants';
 import Image from 'next/image';
 import Link from 'next/link';
 
 import { styles as articleStyles } from './article-tile.style';
 import { ArticleTileProps } from './article-tile.types';
 
+const SIZES = {
+  [BREAKPOINTS.xl]: '50vw, 33vw',
```

Comment:
Not related to CSP, is this an intentional change?

Reply:
Yes it was showing warnings on the console regarding performance.